gmaps/docs/guides/01-getting-started.md

```diff
@@ -0,0 +1,171 @@
+
+### Define credentials
+
+In order to render data from CARTO you need to have a CARTO account and then get the necessary [credentials](/developers/fundamentals/authorization/).
+
+The first thing you need to do is authenticate the client with the `deck.carto.setDefaultCredentials` function, using your own `username` and `apiKey`. For guides and examples, we use the `public` CARTO account so you can try out the library:
+
+```js
+deck.carto.setDefaultCredentials({
+    username: "public",
+    apiKey: "default_public",
+});
+```
+
+### Create map layer
+
+Now you can add a map layer from the public account. In order to add the layer, we will use the [`GoogleMapsOverlay`](https://deck.gl/docs/api-reference/google-maps/google-maps-overlay) class from deck.gl. We need to specify the layers as an array. In this case, we only have one [`CartoSQLLayer`](https://deck.gl/docs/api-reference/carto/carto-sql-layer) from the deck.gl CARTO submodule. We pass three parameters to the constructor:
+
+- `data` contains the dataset that we want to visualize. It can be the name of a dataset or a SQL query. For best performance, we recommend to use always a SQL query retrieving only the fields we need in the client. Otherwise we could be generating very large tiles that will degrade the performance.
+
+- Style parameters. In order to display the information the [`CartoSQLLayer`](https://deck.gl/docs/api-reference/carto/carto-sql-layer) uses the deck.gl [`MVTLayer`](https://deck.gl/docs/api-reference/geo-layers/mvt-layer) that takes advantage of [`GeoJsonLayer`](https://deck.gl/docs/api-reference/layers/geojson-layer) to style the features, so all the properties of this class are supported. The style parameters depend on the dataset type of geometry. In this case, we are adding a point layer, so we specify the point color using the `getFillColor` property and the point size through the `pointRadiusMinPixels` property.
+   
+```js
+const deckOverlay = new deck.GoogleMapsOverlay({
+    layers: [
+        new deck.carto.CartoSQLLayer({
+            data: `SELECT the_geom_webmercator, name FROM populated_places`,
+            getFillColor: [238, 77, 90],
+            pointRadiusMinPixels: 2.5
+        }),
+    ]
+});
+```
+
+### Add map layer
+
+Finally you need to use the [`setMap`] function on the [`GoogleMapsOverlay`](https://deck.gl/docs/api-reference/google-maps/google-maps-overlay) object to add the layer to the map.
+
+```js
+deckOverlay.setMap(map);
+```
+
+### All together
+
+<div class="example-map">
+    <iframe
+        id="getting-started-step-2"
+        src="/developers/gmaps/examples/maps/guides/getting-started/step-2.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
+> You can explore the final step [here](/developers/gmaps/examples/maps/guides/getting-started/step-2.html)
```

Comment:
```suggestion
> You can explore the final step [here](/developers/gmaps/examples/maps/guides/step-2.html)
```

Reply:
I've moved the guide examples to the right folder